TLM/TLM/Manager/Impl/VehicleBehaviorManager.cs

```diff
@@ -1549,62 +1549,61 @@ IJunctionRestrictionsManager junctionRestrictionsManager
                             if (logPriority) {
                                 Log._Debug(
                                     $"VehicleBehaviorManager.MayChangeSegment({frontVehicleId}): " +
-                                      $"{sign} sign. waittime={extVehicle.waitTime}");
+                                    $"{sign} sign. waittime={extVehicle.waitTime}");
```

Comment:
This would be better as `Log._DebugFormat()`?

Reply:
Ok, I can't change it to `DebugIf` because extVehicle is `ref` variable and reference variable cannot be used in lambda expressions.